grammars/go.cson

```diff
@@ -85,7 +85,7 @@
   }
   {
     'name': 'storage.type.go'
-    'match': '\\b(chan|map|bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
+    'match': '\\b(bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
```

Comment:
It is true that `chan` and `map` are _keywords_ used to define _types_; your opinion isn't unfounded.

However, the specification does list them as keywords. Additionally, all other type related keywords `type`, `struct`, `interface` and `func` are arguably also used to define types, but they aren't marked with the `storage.type.go` scope; they have the `keyword.go` scope. So should `chan` and `map`.